build.cake

```diff
@@ -249,26 +248,13 @@ Task("RunUnitTests")
         .ExcludeByFile("*/*.g.cs")
         .ExcludeByFile("*/*.g.i.cs")
         .ExcludeByFile("*splat/splat*")
-        .ExcludeByFile("*ApprovalTests*"));
+        .ExcludeByFile("*ApprovalTests*");
 
-    OpenCover(testFodyAction,
-        testCoverageOutputFile,
-        new OpenCoverSettings {
-            ReturnTargetCodeOffset = 0,
-            ArgumentCustomization = args => args.Append("-mergeoutput")
-        }
-        .WithFilter("+[*]*")
-        .WithFilter("-[*.Testing]*")
-        .WithFilter("-[*.Tests*]*")
-        .WithFilter("-[ReactiveUI.Events]*")
-        .WithFilter("-[Splat*]*")
-        .WithFilter("-[ApprovalTests*]*")
-        .ExcludeByAttribute("*.ExcludeFromCodeCoverage*")
-        .ExcludeByFile("*/*Designer.cs")
-        .ExcludeByFile("*/*.g.cs")
-        .ExcludeByFile("*/*.g.i.cs")
-        .ExcludeByFile("*splat/splat*")
-        .ExcludeByFile("*ApprovalTests*"));
+    OpenCover(tool => RunTests(tool, "ReactiveUI.Tests"), testCoverageOutputFile, openCoverSettings);
+    OpenCover(tool => RunTests(tool, "ReactiveUI.Fody.Tests"), testCoverageOutputFile, openCoverSettings);
+    // TODO: seems the leak tests never worked as part of the CI, fix. For the moment just make sure it compiles.
```

Comment:
Is it worth raising an issue for this bit?

Reply:
Probably. Will do it in a sec